x-pack/plugins/security_solution_serverless/public/navigation/navigation_tree.ts

```diff
@@ -59,7 +59,9 @@ export const getFormatChromeProjectNavNodes = (services: Services) => {
       const navLinkId = getNavLinkIdFromProjectPageName(id);
 
       if (chrome.navLinks.has(navLinkId)) {
-        const breadcrumbHidden = HIDDEN_BREADCRUMBS.has(id);
+        const breadcrumbHidden =
+          HIDDEN_BREADCRUMBS.has(id) ||
+          id.startsWith('management:'); /* management sub-pages set their breadcrumbs themselves */
```

Comment:
alternatively, we could list all the management subpages in the HIDDEN_BREADCRUMBS, but even though this looks hacky, I thought this is more reliable in case new management subpage is added

Reply:
Yes, that's okay. 
But would it be possible to extract these conditions into a `isBreadcrumbHidden = (id: ProjectPageName): boolean` function? (it is okay in this same file just after the `HIDDEN_BREADCRUMBS` definition) 
This way we simplify the _reduce_ function, we can check it directly in the `link` creation:
```
...(isBreadcrumbHidden(id) && { breadcrumbStatus: 'hidden' }),
```